utils/permissions.py

```diff
@@ -14,4 +17,17 @@ class IsEventOwner(BasePermission):
     """
 
     def has_object_permission(self, request, view, obj):
-        return obj.event.user == request.user
\ No newline at end of file
+        return obj.event.user == request.user
+    
+class isPremiumUser(BasePermission):
```

Comment:
to maintain consistency you can change it to `IsPremiumUser` (capital i)

Reply:
Oh right, nice catch! Thank you for the suggestion, I have changed it.